src/community/jms-cluster/jms-geoserver/src/main/java/org/geoserver/cluster/impl/handlers/DocumentFileHandler.java

```diff
@@ -28,12 +27,12 @@ public DocumentFileHandler(XStream xstream, Class clazz) {
 
 	@Override
 	public boolean synchronize(DocumentFile event) throws Exception {
-		FileOutputStream fout = null;
+		OutputStream fout = null;
```

Comment:
Please convert these tabs to spaces. Can this be converted to a Java 7 try-with-resources?